extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/services.js

```diff
@@ -6,8 +6,8 @@ angular.module('PaperUI.services', [ 'PaperUI.constants' ]).config(function($htt
     $httpProvider.interceptors.push(function($q, $injector) {
         return {
             'responseError' : function(rejection) {
-                var showError = !!rejection.showError;
-                if (showError) {
+                var showError = rejection.showError;
+                if (showError !== false) {
```

Comment:
Error should be shown when showError is not set i.e. when it is undefined. When rejection.showError is undef first negation returns true and second one converts it to false and the error is not shown. The error should only be shown when showError is EXACTLY equal to false. i.e. when it is set.